packages/vite-app/vite.config.mjs

```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from 'vite';
+import { extensions, classicEmberSupport, ember } from '@embroider/vite';
+import { babel } from '@rollup/plugin-babel';
+
+export default defineConfig({
+  plugins: [
+    classicEmberSupport(),
+    ember(),
+    // extra plugins here
+    babel({
+      babelHelpers: 'runtime',
+      extensions,
+    }),
+  ],
+  // build: {
+  //   target: "ES2022"
+  // },
+  optimizeDeps: {
+    esbuildOptions:{
+      target: 'ES2022'
```

Comment:
we talked about adding this upstream btw, to support the exact use case that you're targeting 👍